### DISCUSSION TITLE
Different setting for IPF plot

### DISCUSSION THREAD

**Discussion by NovakEvana** [mtex-toolbox/mtex#1740] [Ask Anything] [answered]
Dear all,

We're usually looking at an Inverse pole figure map following the ND, TD, or RD directions of the sample coordinate. So how to config the IPF through an angle corresponding to one of those axes? For example, instead of IPF-ND, is that possible to change it to IPF-60 degree to ND? I know there is something to do with `vector3d `but not sure how to change it.

Thanks.

**Comment by kilir** [comment, marked as answer]
Hi,
an `ipfColorkey` such as the `ipfHSVKey` comes with a property called `inversePoleFigureDirection` which can usually set to any vector3d required.
```
ck=ipfHSVKey(cs)
ck.inversePoleFigureDirection=vector3d(2,1,3)
```
Cheers,
Rüdiger

**Reply by kilir** [reply]
Either compute the corresponding x,y,z coordinates or more simply, you can define a vector3d also using `vector3d.byPolar()`, which takes polar and azimuthal angles of polar coordinates.
You can always plot those to be sure you get what you want. 
See help vector3d.
Hope that helps.
Cheers,
Rüdiger

**Reply by NovakEvana** [reply]
Thanks Rudiger so much!
